Trader.Client/Views/LogEntryViewer.cs

```diff
@@ -99,11 +99,11 @@ public LogEntryViewer(ILogEntryService logEntryService)
 
```

Comment:
Thanks for taking the time to explain. 
My one remaining confusion is over master branch vs RolandPheasant\master branch. Are  they one and the same? I thought I had messed up when I didn't push the changes to my fork but rather just made the pull request to you directly after my changes. 

When I'm making changes in a forked repo (stroked\master in this case), I'm really just trying to push the changes to you, then why is it called "pull request"? Seems unintuitive. Althought, from your point-of-view it makes sense because I'm asking you to pull in my changes into your repo...so pull-request seems appropriate.